src/librustc_typeck/check/method/probe.rs

```diff
@@ -183,7 +183,7 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
                scope_expr_id);
         let method_names =
             self.probe_op(span, mode, None, Some(return_type), IsSuggestion(true),
-                          self_ty, scope_expr_id, ProbeScope::TraitsInScope,
+                          self_ty, scope_expr_id, ProbeScope::AllTraits,
```

Comment:
Should we do this if and only if `TraitsInScope` doesn't yield any suggestions?

Reply:
With no other callers, I feel OK about changing this.